### ISSUE TITLE
Crash after logos

### ISSUE THREAD

**Issue by aliak629918** [mi5hmash/LimebrellaSharp#6]
after resigning and copying in my new savefile the game crashes after displaying the game logos. Not Sure why.

**Comment by HytonightYX** [comment]
same too

**Comment by mi5hmash** [comment]
Hello guys. I am out of home right now. Do you have Discord? Maybe we can catch there later?

**Comment by aliak629918** [comment]
@mi5hmash my discord is nothisispatrick2040

**Comment by mi5hmash** [comment]
Okay, so I have successfully helped the @aliak629918.
@HytonightYX, please contact me on Discord or send me your SaveData files with the old SteamID and a new SteamID and I will try to resign them the same way I always do. If you do not have Discord then send me a private message to my email address: mike.gebicki@gmail.com.

**Comment by FarshadGrayson** [comment]
I have the same issue , can we get a video tutorial on how to use this tool ? seems like we are doing it wrong

**Comment by mi5hmash** [comment]
I will when I find some spare time to do so. :)

**Comment by FarshadGrayson** [comment]
> @FarshadGrayson I will when I find some spare time to do so. :)
> 
> When you wish to resign the SaveData files, you need to drag and drop the file onto the "Input File Path", then type your old SteamID number in the left field, and the new one in the field to its right. After that you hit "Resign All" button and you should be able to find resigned files inside the "_OUTPUT" folder, located in the same directory where the LimebrellaSharp.exe is. These files are ready to go.

Thank you ! I used to unpack , resign and then pack all , Now I only used Resign All option and it worked !

**Comment by freejool** [comment]
> @FarshadGrayson I will when I find some spare time to do so. :)
> 
> When you wish to resign the SaveData files, you need to drag and drop the file onto the "Input File Path", then type your old SteamID number in the left field, and the new one in the field to its right. After that you hit "Resign All" button and you should be able to find resigned files inside the "_OUTPUT" folder, located in the same directory where the LimebrellaSharp.exe is. These files are ready to go.

Did what you say. But I can only see the "professional" option on "new game" because I have beat the game once. There is no save points for me to load. 

it seems that only the main data was repacked successfully but not the save slots data.

**Comment by mi5hmash** [comment]
So in the end, I hid the "Unpack All" and "Pack All" buttons by default as they are for the narrower spectrum of users. The majority of people simply want to resign their SaveData files, and the "Resign All" button is all they need to achieve their goal. However, the other buttons I mentioned above, can still be activated by rapidly clicking on the version label, three times in a row.

The other thing I have implemented is a log-creation feature that records operations performed on SaveData files and may help figure out what went wrong.

Finally, for people who do not see their new SaveData files in the Load Game menu despite they resigned them properly, I wrote a guide on [how to unlock SaveData slots](https://github.com/mi5hmash/LimebrellaSharp/tree/main/.resources/Save%20Files).

Hope, that all these precautions will prevent this kind of issue from happening again and I am closing this issue.

Still, if anyone needs help, they can open a new thread or contact me on Discord. Happy New Year! :)
